runtime-modules/content/src/benchmarks/benchmarking.rs

```diff
@@ -0,0 +1,226 @@
+        assert!(group == CuratorGroup::create(true, &permissions_by_level));
+        assert_last_event::<T>(Event::<T>::CuratorGroupCreated(group_id).into());
+    }
+
+    update_curator_group_permissions {
+        let a in 0 .. (T::MaxKeysPerCuratorGroupPermissionsByLevelMap::get() as u32);
+
+        let lead_account = insert_content_leader::<T>();
+        let group_id = setup_worst_case_curator_group_with_curators::<T>(
+            T::MaxNumberOfCuratorsPerGroup::get()
+        )?;
+        let permissions_by_level: ModerationPermissionsByLevel::<T> = (0..a).map(
+            |i| (i.saturated_into(), worst_case_content_moderation_actions_set())
+        ).collect();
+    }: _ (
+        RawOrigin::Signed(lead_account),
+        group_id,
+        permissions_by_level.clone()
+    )
+    verify {
+        let group = Pallet::<T>::curator_group_by_id(group_id);
+        assert_eq!(group.get_permissions_by_level(), &permissions_by_level);
+        assert_last_event::<T>(Event::<T>::CuratorGroupPermissionsUpdated(
+            group_id,
+            permissions_by_level
+        ).into());
+    }
+
+    set_curator_group_status {
+        let lead_account = insert_content_leader::<T>();
+        let group_id = setup_worst_case_curator_group_with_curators::<T>(
+            T::MaxNumberOfCuratorsPerGroup::get()
+        )?;
+        let group = Pallet::<T>::curator_group_by_id(group_id);
+        assert!(group.is_active());
+    }: _ (
+        RawOrigin::Signed(lead_account),
+        group_id,
+        false
+    )
+    verify {
+        let group = Pallet::<T>::curator_group_by_id(group_id);
+        assert!(!group.is_active());
+        assert_last_event::<T>(Event::<T>::CuratorGroupStatusSet(group_id, false).into());
+    }
+
+    add_curator_to_group {
+        let lead_account = insert_content_leader::<T>();
+        let permissions = worst_case_channel_agent_permissions();
+        let group_id = setup_worst_case_curator_group_with_curators::<T>(
+            T::MaxNumberOfCuratorsPerGroup::get() - 1
+        )?;
+        let (curator_id, _) = insert_curator::<T>(
+            CURATOR_IDS[T::MaxNumberOfCuratorsPerGroup::get() as usize - 1]
+        );
+        let group = Pallet::<T>::curator_group_by_id(group_id);
+        assert_eq!(group.get_curators().get(&curator_id), None);
+    }: _ (
+        RawOrigin::Signed(lead_account),
+        group_id,
+        curator_id,
+        permissions.clone()
+    )
+    verify {
+        let group = Pallet::<T>::curator_group_by_id(group_id);
+        assert_eq!(group.get_curators().get(&curator_id), Some(&permissions));
+        assert_last_event::<T>(Event::<T>::CuratorAdded(group_id, curator_id, permissions).into());
+    }
+
+    remove_curator_from_group {
+        let lead_account = insert_content_leader::<T>();
+        let group_id = setup_worst_case_curator_group_with_curators::<T>(
+            T::MaxNumberOfCuratorsPerGroup::get()
```

Comment:
I think It's better to add this, as you cannot create more curators than the min of these two limits (workers and curators), this can differ later in runtime and fail the test.
```suggestion
min(<T as working_group::Config<ContentWorkingGroupInstance>>::MaxWorkerNumberLimit::get(),
        T::MaxNumberOfCuratorsPerGroup::get()),
```